### PR TITLE
Deprecate `n_splits` with `cv`

### PR DESCRIPTION
**Pull request by vsyrgkanis**

### CODE REVIEW

File: doc/spec/estimation/dml.rst [hunk 1]
```diff
@@ -551,7 +551,7 @@ Usage FAQs
     If one uses cross-validated estimators as first stages, then model selection for the first stage models
     is performed automatically.
 
-- **How should I set the parameter `n_splits`?**
+- **How should I set the parameter `cv`?**
 
     This parameter defines the number of data partitions to create in order to fit the first stages in a
     crossfittin manner (see :class:`._OrthoLearner`). The default is 2, which
```
Comment (kbattocchi) [inline doc/spec/estimation/dml.rst:557]:
```suggestion
    crossfitting manner (see :class:`._OrthoLearner`). The default is 2, which
```

**Review by kbattocchi — APPROVED** [review]
Found a few typos unrelated to your changes, but otherwise looks good.

File: doc/spec/estimation/dml.rst (hunk 1 above)
Comment (vsyrgkanis) [reply doc/spec/estimation/dml.rst:557]: Done

File: doc/spec/estimation/dr.rst [hunk 2]
```diff
@@ -467,7 +467,7 @@ Usage FAQs
     If one uses cross-validated estimators as first stages, then model selection for the first stage models
     is performed automatically.
 
-- **How should I set the parameter `n_splits`?**
+- **How should I set the parameter `cv`?**
 
     This parameter defines the number of data partitions to create in order to fit the first stages in a
     crossfittin manner (see :class:`._OrthoLearner`). The default is 2, which
```
[the remark this answers is not in the document]
Comment (vsyrgkanis) [reply doc/spec/estimation/dr.rst:473]: Done

**Pull request merged**